picker-starter/src/core/types.ts

```diff
@@ -31,13 +31,13 @@ export type CursorPaginatedResult<Item> = {
 
 export type PickerServiceValidationResult =
   | {
-      exampleOptions: Record<string, string>
+      exampleOptions?: Record<string, string>
       value: undefined
       error: string
     }
   | {
-      exampleOptions: Record<string, string>
-      value: PickerService
+      exampleOptions?: Record<string, string>
+      value: unknown
```

Comment:
I'll need to understand why the type checking is accusing error while using the `PickerService` type.  I'm going to solve it before opening the PR.